hail/python/hailtop/pipeline/task.py

```diff
@@ -373,7 +373,7 @@ def cpu(self, cores):
             Same task object with CPU requirements set.
         """
 
-        self._cpu = cores
+        self._cpu = str(cores)
```

Comment:
what's the significance of these two changes?

Reply:
It seems like we should parse these numbers into integers in the smallest unit in our system, not store them as strings

Reply:
The reason for the two changes is the documentation says we accept str, float, or int. However, the batch client is expecting a string.